src/Microsoft.OData.Edm/ExtensionMethods/EnumHelper.cs

```diff
@@ -126,6 +126,48 @@ public static bool TryParseEnum(this IEdmEnumType enumType, string value, bool i
             return true;
         }
 
+        /// <summary>
+        /// Parse an enum integral value to enum member.
+        /// </summary>
+        /// <param name="enumType">edm enum type</param>
+        /// <param name="value">input integral value.</param>
+        /// <param name="enumMember">parsed result.</param>
+        /// <returns>true if parse succeeds, false if parse fails.</returns>
+        public static bool TryParseEnum(this IEdmEnumType enumType, long value, out IEdmEnumMember enumMember)

```

Comment:
If you used `TryParse`, would it conflict with existing framework `TryParse` methods? If not, `TryParse` is a better fit than `TryParseEnum`

Reply:
@gathogojr 

I have renamed the method to `TryParse`. Yes, you are right. It won't cause any conflict